scripts/provision.sh

```diff
@@ -200,7 +201,41 @@ start_datastore() {
 }
 
 start_minio() {
-  sudo hab svc load "${BLDR_ORIGIN}/builder-minio" --channel "${BLDR_MINIO_CHANNEL:=$BLDR_CHANNEL}" --force
+  
+  set +e
+  is_minio_migration_needed
+  migration_needed=$?
+  set -e
+  
+  if [ "$migration_needed" -eq 1 ]; then
+    sudo hab svc load "${BLDR_ORIGIN}/builder-minio" --channel "stable" --force
+    bash ./minio-update.sh preflight_checks
+    echo MinIO migration required
+    backup_minio_data
+    echo starting minio
```

Comment:
This message does not look right. The following code downloads the data and stops it.